src/common/util/src/file_util.cpp

```diff
@@ -52,7 +52,14 @@
 
 std::string ov::util::get_file_name(const std::string& s) {
     std::string rc = s;
+    // Linux-style separator
     auto pos = s.find_last_of('/');
```

Comment:
can we reuse https://github.com/openvinotoolkit/openvino/blob/1e448528d9d02d9859c3eea3ef4751443cc11201/src/common/util/include/openvino/util/file_util.hpp#L22-L28 to make the code more universal?

Reply:
I copied this from similar code located in (including in comments):
https://github.com/openvinotoolkit/openvino/blob/1e448528d9d02d9859c3eea3ef4751443cc11201/src/common/util/src/file_util.cpp#L80-L95